modules/likes.php

```diff
@@ -557,4 +557,26 @@ function admin_bar_likes() {
 	}
 }
 
+/**
+ * Add Likes post_meta to the REST API Post response.
+ *
+ * @action rest_api_init
+ * @uses register_meta
+ */
+function post_likes_register_meta() {
+	register_meta(
+		'post', 'switch_like_status',
+		array(
+			'type'			=> 'boolean',
+			'single'		=> true,
+			'show_in_rest'	=> true,
+		)
+	);
+}
+
+// Add Likes post_meta to the REST API Post response.
+if ( function_exists( 'register_meta' ) ) {
```

Comment:
is this a special instance where we're checking for `register_meta`? if so, could we add an explanatory comment why in some cases it's not there? (unless it would be obvious to most developers who work in Jetpack/WP)

Reply:
No idea - I took this from the tutorial. Maybe it dates from when the API was a plugin?

Reply:
Alright - I guess it's not [here](https://wordpress.org/gutenberg/handbook/designers-developers/developers/tutorials/plugin-sidebar-0/plugin-sidebar-3-register-meta/) so I'm okay with removing the guard if you don't think it's needed.